server/sonar-web/src/main/js/components/SelectList/index.js

```diff
@@ -392,12 +392,12 @@ window.SelectList = function(options) {
  * SelectList API Methods
  */
 
-window.SelectList.prototype.filter = function(filter) {
+SelectList.prototype.filter = function(filter) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Declare a "SelectList" class and move this declaration of "filter" into it. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3525)